File: BACKEND/PickleballCourtBookingSystem/PickleballCourtBookingSystem.Core/Entities/Booking.cs
using PickleballCourtBookingSystem.Core.PEnum;

namespace PickleballCourtBookingSystem.Core.Entities

{
    public class Booking
    {
        public Guid? Id { get; set; }
        public DateTime? TimeBooking { get; set; }
        public double? Amount { get; set; }
        public BookingStatusEnum? Status { get; set; }
        public PaymentStatusEnum? PaymentStatus { get; set; }
        public Guid? CourtId { get; set; }
        public Guid? CustomerId { get; set; }
        public Guid? CourtClusterId { get; set; }
        public Guid? CourtOwnerId { get; set; }
    }
}

File: AndroidApp/app/src/main/java/com/maxholmes/androidapp/data/model/Booking.kt
package com.maxholmes.androidapp.data.model

import android.os.Parcelable
import com.maxholmes.androidapp.utils.enum.BookingStatusEnum
import com.maxholmes.androidapp.utils.enum.PaymentStatusEnum
import kotlinx.parcelize.Parcelize
import java.time.LocalDateTime

@Parcelize
data class Booking(
    var id: String,
    var timeBooking: String,
    var status: BookingStatusEnum,
    var amount: Double,
    var paymentStatus: PaymentStatusEnum,
    var courtId: String,
    var courtClusterId: String,
    var customerId: String
) : Parcelable
